### PR TITLE
Add -UseBasicParsing to all powershell Invoke-WebRequest calls to avoid dependency on IE

### PR DESCRIPTION
**Pull request by dannyvv**
Without this flag PowerShell relies on IE to perform downloads. On server images with IE security enhancements this will fail unless they are disabled....

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/6088)

### CODE REVIEW

**Comment by NickGerleman** [comment]
FYI that the current PR I have out adds another one of these 😛. If it goes through another round I will add this though.

**Comment by dannyvv** [comment]
> FYI that the current PR I have out adds another one of these 😛. If it goes through another round or you merge first I will add this though.

No worries, I'll hold off until after your PR.

File: vnext/Microsoft.ReactNative/getLayoutProps.ps1 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # This script produces the values for the layoutProperties vector
 # in TestHook.cpp
-$page = Invoke-WebRequest "https://reactnative.dev/docs/layout-props"
+$page = Invoke-WebRequest -UseBasicParsing "https://reactnative.dev/docs/layout-props"
```
Comment (NickGerleman) [inline vnext/Microsoft.ReactNative/getLayoutProps.ps1:3]: This doesn't hurt, but this looks like a one-off script to produce these. I'm not actually sure we should have this file at all.

**Review by NickGerleman — APPROVED** [review]

**Review by asklar — APPROVED** [review]

File: vnext/Microsoft.ReactNative/getLayoutProps.ps1 (hunk 1 above)
Comment (dannyvv) [reply vnext/Microsoft.ReactNative/getLayoutProps.ps1:3]: I"ll leave as-is.

**Comment by JunielKatarn** [comment]
Going forward, should we just move to PowerShellCore (pwsh.exe)?
It would add another install requirement, but long term it will likely be the default Windows shell.

I think PS Core doesn't have any IE dependencies.

**Comment by dannyvv** [comment]
> Going forward, should we just move to PowerShellCore (pwsh.exe)?
> It would add another install requirement, but long term it will likely be the default Windows shell.
> 
> I think PS Core doesn't have any IE dependencies.

I would like that, but unfortunately our powershell scripts run as part of the customers' builds, so that means all developers building RNW apps would also need to have pwsh installed, and the right versions depending on the features we use...
I personally think it is okay to standardize on the one shipped in windows for now.

**Comment by asklar** [comment]
> > Going forward, should we just move to PowerShellCore (pwsh.exe)?
> > It would add another install requirement, but long term it will likely be the default Windows shell.
> > I think PS Core doesn't have any IE dependencies.
> 
> I would like that, but unfortunately our powershell scripts run as part of the customers' builds, so that means all developers building RNW apps would also need to have pwsh installed, and the right versions depending on the features we use...
> I personally think it is okay to standardize on the one shipped in windows for now.

Parts of the UWP tooling expect to run in windows powershell not pwsh, at least for some versions of VS, so we're stuck for a while. Some of it is fixed now but I expect other pieces of UWP project tooling to have the same kinds of issues: https://github.com/PowerShell/PowerShell/issues/12739

**Pull request merged**